app/assets/javascripts/components/students/student_list.jsx

```diff
@@ -18,7 +18,7 @@ import { notifyOverdue } from '../../actions/server_actions_redux.js';
 import CourseUtils from '../../utils/course_utils.js';
 import { getFiltered } from '../../utils/model_utils.js';
 
-export const StudentList = createReactClass({
+export const ConnectedStudentList = createReactClass({
```

Comment:
The naming is backwars here. This is the unconnected component. The connected one is the default export at the end of the file.

If you leave this as StudentList, then you can do `import ConnectedStudentList` in StudentsHandler (and change the usage of it in line 60 to match).

Reply:
Yeah, so sorry. I didn't realize this. Can you make this change as a maintainer?